HumanPlayer.js

```diff
@@ -0,0 +1,46 @@
+// const readline = require('readline-sync');
+
+class HumanPlayer {
+  constructor(name) {
+    this.name = name;
+  }
+  getLetter(){
+
+    return readline.question("Please enter your guess: ")
+}
+
+  // getMove() {
```

Comment:
Removing unused code before submitting is something you should get into the habit of doing since it effects your grade.